backend/app/core/config.py

```
@@ -74,6 +74,29 @@ def assemble_allowed_hosts(cls, v):
+        """SECRET_KEYのセキュリティ検証"""
+        dangerous_values = [
+            "your-super-secret-key-here",
+            "your-super-secret-key-change-this-in-production",
+            "secret",
+            "changeme",
+            "default"
+        ]
+        
+        if v.lower() in [val.lower() for val in dangerous_values]:
+            raise ValueError(
+                f"SECRET_KEYにデフォルト値が設定されています。"
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)